src/Test.UnitTests.Sarif.Driver/TestRule.cs

```diff
@@ -245,11 +256,15 @@ public static void RaiseExceptionViaReflection()
 
         public IEnumerable<IOption> GetOptions()
         {
-            return new IOption[] { Behaviors, UnusedOption };
+            return new IOption[] { Behaviors, UnusedOption, ErrorsCount };
         }
 
         private const string AnalyzerName = TestRuleId + "." + nameof(TestRule);
 
+        public static PerLanguageOption<uint> ErrorsCount { get; } =
+            new PerLanguageOption<uint>(
+                AnalyzerName, nameof(ErrorsCount), defaultValue: () => { return 15; });
```

Comment:
>15 [](http://example.com/codeflow?start=80&length=2)

Restore this to 1, that was the previous behavior. Or provide a justification for the updated value.